shared/js/background/dnr-session-rule-id.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * For managing dynamically created MV3 session rules
+ * getNextSessionRuleId will return the next unique session rule id to use when creating new session rules
+ **/
+const browserWrapper = require('./wrapper.es6')
+const SESSION_RULE_ID_START = 100000
+const SESSION_RULE_STORAGE_KEY = 'sessionRuleOffset'
+let sessionRuleOffset = 0
+browserWrapper.getFromSessionStorage(SESSION_RULE_STORAGE_KEY).then(offset => {
+    if (offset) {
+        sessionRuleOffset = offset
+    }
+})
+
+export function getNextSessionRuleId () {
```

Comment:
Please can you add a JSDoc comment and unit tests for this? We're likely going to rely on it for Click to Load as well.